polkadot/node/subsystem-bench/src/subsystem-bench.rs

```diff
@@ -165,6 +184,11 @@ impl BenchCli {
 		env.runtime()
 			.block_on(availability::benchmark_availability_read(&mut env, state));
 
+		if let Some(agent_running) = agent_running {
```

Comment:
This code has changed, you should update the branch and make sure we still run it before exit.

Reply:
Ok